Sources/MediaViewController.swift

```diff
@@ -53,6 +54,7 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
         setupCollectionView()
         setupSearchController()
         setupNavigationBar()
+        setUpEmptyBackgroundView()
```

Comment:
`setUp` vs `setup`

I do not have any preference among them, but just because of karma I think they should be all the same.

Reply:
Oh I didn't even notice that. Thanks @dcordero